NPMPack.ps1

```diff
@@ -23,20 +16,11 @@ if($sdkVersion){
 }
 
 npm install
-
-
-
-Switch ("$env:Build_SourceBranchName") 
-{
-    "develop"{		
-		npm --% publish --registry=%NPMFeed% --tag %tag%	
-	}
-
-	"master" {		
-		$env:version = $json.version
-		$env:name = $json.name
-		npm --% dist-tag add %name%@%version% %tag% --registry=%NPMFeed%
-	}
-}
+		
+npm --% publish --registry=%NPMFeed% 	
+	
+$env:version = $json.version
+$env:name = $json.name
+npm --% dist-tag add %name%@%version% --registry=%NPMFeed%
```

Comment:
This part is not needed anymore since we don't need to add the tag to the package. The env:version and env:name lines can also be removed